### PR TITLE
STY POC - Cps images

### PR DESCRIPTION
**Pull request by chris-hinds**
**Overall change:** 

Creates a new images block in mediaContainer.

Imports and uses the story promo image for now as the main image container is tightly coupled with optimo blocks

image

### CODE REVIEW

File: src/app/containers/MediaPageMain/blocks/image.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { StoryPromoImage } from '../../StoryPromo/index.jsx'
+
+// eslint-disable-next-line react/prop-types
+const HeadingContainer = ({ uuid, path, height, width }) => {
```
Comment (dr3) [inline src/app/containers/MediaPageMain/blocks/image.jsx]:
```suggestion
const ImageContainer = ({ uuid, path, height, width }) => {
```

File: src/app/containers/MediaPageMain/blocks/image.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    const imageValues = {height, width, path}
+    return (
+        <StoryPromoImage key={uuid} topStory="false" imageValues={imageValues} lazyLoad="true"/>
+    )
+    };
+
+export default HeadingContainer;
```
Comment (dr3) [inline src/app/containers/MediaPageMain/blocks/image.jsx]:
```suggestion
export default ImageContainer;
```

File: src/app/containers/MediaPageMain/blocks/image.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { StoryPromoImage } from '../../StoryPromo/index.jsx'
+
+// eslint-disable-next-line react/prop-types
+const HeadingContainer = ({ uuid, path, height, width }) => {
+    const imageValues = {height, width, path}
+    return (
+        <StoryPromoImage key={uuid} topStory="false" imageValues={imageValues} lazyLoad="true"/>
+    )
+    };
```
Comment (dr3) [inline src/app/containers/MediaPageMain/blocks/image.jsx]:
```suggestion
};
```

File: src/app/containers/MediaPageMain/blocks/image.jsx (hunk 1 above)
Comment (chris-hinds) [reply src/app/containers/MediaPageMain/blocks/image.jsx]: lol completely forgot to change those, gotta love copy and paste

**Review by dr3 — APPROVED** [review]

**Pull request merged**